modules/Bio/EnsEMBL/Analysis/Hive/Config/GenblastHomology.pm

```diff
@@ -0,0 +1,532 @@
+    'taxon_id'                  => '', # should be in the assembly report file
+    'uniprot_set'               => '', # e.g. mammals_basic, check UniProtCladeDownloadStatic.pm module in hive config dir for suitable set,
+    'output_path'               => '', # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+
+########################
+# Pipe and ref db info
+########################
+
+    # The following might not be known in advance, since the come from other pipelines
+    # These values can be replaced in the analysis_base table if they're not known yet
+    # If they are not needed (i.e. no projection or rnaseq) then leave them as is
+
+    'pipe_db_name'                  => $self->o('dbowner').'_'.$self->o('production_name').'_pipe_'.$self->o('release_number'),
+    'dna_db_name'                   => $self->o('dbowner').'_'.$self->o('production_name').'_core_'.$self->o('release_number'),
+
+    'genblast_db_server'           => $self->o('databases_server'),
+    'genblast_db_port'             => $self->o('databases_port'),
+
+
+    'killlist_db_server'           => $self->o('databases_server'),
+    'killlist_db_port'             => $self->o('databases_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release'              => $ENV{ENSEMBL_RELEASE}, # this is the current release version on staging to be able to get the correct database
+
+    databases_to_delete => ['genblast_db'],
+
+########################
+# BLAST db paths
+########################
+    'base_blast_db_path'        => $ENV{BLASTDB_DIR},
+
+######################################################
+#
+# Mostly constant settings
+#
+######################################################
+
+    genome_dumps                  => catdir($self->o('output_path'), 'genome_dumps'),
+    # This one is used by most analyses that run against a genome flatfile like exonerate, genblast etc. Has slice name style headers. Is softmasked
+    softmasked_genome_file        => catfile($self->o('genome_dumps'), $self->o('species_name').'_softmasked_toplevel.fa'),
+    # This one is used in replacement of the dna table in the core db, so where analyses override slice->seq. Has simple headers with just the seq_region name. Also used by bwa in the RNA-seq analyses. Not masked
+
+    'min_toplevel_slice_length'   => 250,
+
+    'homology_models_path'        => catdir($self->o('output_path'),'homology_models'),
+
+    ensembl_analysis_script           => catdir($self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts'),
+    remove_small_orf_script             => catfile($self->o('ensembl_analysis_script'), 'genebuild', 'remove_small_orf.pl'),
+  
+    # cutoffs for removing small_orf genes
+    'small_orf_cutoff' => '100',
+    'intron_cutoff' => '75',
```

Comment:
This can be removed, it is not used in the config

Reply:
Requested changed effected